packages/teleterm/src/ui/StatusBar/ShareFeedback/ShareFeedbackForm.tsx

```diff
@@ -39,9 +39,20 @@ export function ShareFeedbackForm({
         value={formValues.email}
         onChange={e => updateFormField('email', e.target.value)}
       />
+      <FieldInput
+        label="Company name (optional)"
+        autoFocus
+        readonly={disabled}
```

Comment:
We should be using the `disabled` attribute here but that goes beyond the scope of the PR as I see we use it because of the way `FieldInput` is written.